nuxt.config.js

```diff
@@ -228,7 +228,7 @@ export default {
    ** See https://axios.nuxtjs.org/options
    */
   axios: {
-    baseURL: 'https://api.modrinth.com/v2/',
+    baseURL: 'https://staging-api.modrinth.com/v2/',
```

Comment:
Staging

Reply:
It should use staging in development, I accidentally left it with the production URL in the the last PR

Reply:
Vercel builds directly from source, though

Reply:
Vercel & other production environments have an environment variable that override this